lib/rspec/core/hooks.rb

```diff
@@ -13,10 +13,12 @@ module Hooks
       # @overload before(scope, &block)
       #   @param scope [Symbol] `:example`, `:context`, or `:suite`
       #     (defaults to `:example`)
-      # @overload before(scope, conditions, &block)
+      # @overload before(scope, *condition_keys, conditions={}, &block)
```

Comment:
I'm unsure if this is the correct yard syntax, as technically this is not as it is implemented even if its the effect...

Reply:
Picked it up here. Ended up with just using `*conditions`.